datamade/how-to
# add js lib to wagtail template, change gitignore

## Overview

Removed `/lib` from the `.gitignore` file so it gets included. And copied over the `/lib` directory from the `new-django-app` template

Handles #309 

## Testing Instructions

* Pull down changes and navigate to the new-wagtail-app template
* Ensure that `{{cookiecutter.app_name}}/{{cookiecutter.module_name}}/static/js/lib` is present locally
* Ensure that the `.gitignore` present in the wagtail app template does not include `/lib`
